week-5/Homework/mandatory/3-project/README.md

```diff
@@ -9,18 +9,18 @@ Clicking on the buttons should change the "theme" of the website:
 
 - When clicking **blue** it should change:
 
-  - **Jumbotron** background color to `#588fbd`
+  - Div tag with the id **Jumbotron** background color to `#588fbd`
   - **Donate a bike** button background color to `#ffa500`
   - **Volunteer** button background color to `black` and text color to `white`
```

Comment:
Your solution to this exercise is missing. It should be added to `main.js` file in `3-project/js` folder